docs/proposals/algorithms/lifelong-learning/Cloud-Robotic AI Benchmarking for Edge-cloud Collaborative Lifelong Learning.md

```diff
@@ -0,0 +1,239 @@
+# Cloud-Robotic AI Benchmarking for Edge-cloud Collaborative Lifelong Learning
+
+## 1 Motivation
+
+### 1.1 background
+
+It is estimated that by 2025, 75% of the world's data will be generated at the edge, and the computing power on the cloud will be more abundant. Edge-cloud collaborative artificial intelligence will become an inevitable trend, and its demand will be further released. Among them, the global service robot market is expected to reach 90-170 billion US dollars in 2030. The use of cloud-native edge computing and artificial intelligence technology to deal with the issues of the robot industry and complete industrial transformation has also become the focus of the industry.
+
+In recent years, lifelong learning-related algorithms such as Lifelong SLAM and Lifelong Object Detection have become popular for the problem of edge-data heterogeneity and small samples, but the real-world practise requires further considerations on its edge-cloud collaborative nature. To further accelerate research and results transformation, the KubeEdge community released the first open source edge-cloud collaborative lifelong learning framework and its resource orchestration template on KubeEdge-Sedna in June 2021. Moreover, the collaborative AI benchmarking KubeEdge-Ianvs in July 2022 is also released with related benchmark datasets and compute metrics.
+
+This project aims to develop the edge-cloud collaborative lifelong learning benchmarking that are suitable for robotic scenarios based on KubeEdge-Ianvs. This project will help all Edge AI application developers to validate and select the best-matched algorithm of lifelong learning. The benchmark can include dataset, metrics and algorithm. Specific applications include but are not limited to robot navigation, inspection, cleaning, delivery, etc. KubeEdge SIG AI has already prepared real-world datasets for everyone to explore!
+
+### 1.2 Goals
+
+Based on real-world datasets provided by industry members of KubeEdge SIG AI, this project aims to build a lifelong learning benchmarking on KubeEdge-Ianvs. Namely, it aims to help all Edge AI application developers to validate and select the best-matched algorithm of lifelong learning. It includes:
+
+1. Release new datasets for lifelong learning in KubeEdge-Ianvs.
+2. Implement BWT, FWT and other system metrics for lifelong learning in KubeEdge-Ianvs.
+
+## 2 Proposal
+
+This project will provide new real-world data sets for KubeEdge-Ianvs's lifelong learning algorithm, including a public semantic segmentation data set and a data set provided by Huawei. Each dataset released at the same time will have a supporting website to introduce the dataset. In addition, the project will further enrich the metrics method of KubeEdge-Ianvs lifelong learning algorithm, including but not limited to BWT, FWT and other methods. Data sets and metrics methods are described below.
```

Comment:
The public datasets, i.e., cityscapes and synthia, are not new. The description should be corrected.

Reply:
Ok, I will correct it.